src/main/java/ca/ryangreen/apigateway/generic/GenericApiGatewayRequestBuilder.java

```diff
@@ -32,9 +34,18 @@ public GenericApiGatewayRequestBuilder withHeaders(Map<String, String> headers)
         return this;
     }
 
+    public GenericApiGatewayRequestBuilder withParameters(Map<String,List<String>> parameters) {
```

Comment:
Should this be withQueryStringParameters?

Reply:
see above